mocking-stubbing/test/test.js

```diff
@@ -0,0 +1,17 @@
+const { describe, test, before } = require('mocha');
+var { NewContext } = require('typescript-sdk/dist/mock/mock');
+require('typescript-sdk/dist/integrations/node-fetch/require');
+const fetch = require('node-fetch');
+
+describe('Generating of mocks for external dependencies', function () {
+  before(() => {
+    // Setting mode to record before using fetch
+    NewContext({ Mode: 'record', Name: 'testing-auto-mocks' });
+  });
+
+  test('Generating node-fetch mocks', () => {
+    fetch('https://reqres.in/api/users/2')
+      .then((res) => res.text())
+      .then((text) => rs.send(text));
```

Comment:
Please add assertions to check for the expected response.

Reply:
- @re-Tick, done, could you check the new commit?
- off-topic: Currently `NewContext` returns undefined, which makes me unable to check if `Keploy` worked well or not, it'd be nice to have that ability.

Reply:
LGTM.

Reply:
> * @re-Tick, done, could you check the new commit?
> * off-topic: Currently `NewContext` returns undefined, which makes me unable to check if `Keploy` worked well or not, it'd be nice to have that ability.

Yes it makes sense to return an error for failure. If you like to work on this, you can make an issue and propose your changes for discussion.